src/stories/Blocks/event-page/EventPage.tsx

```diff
@@ -1,16 +1,33 @@
 import { FC } from "react";
 import EventHeader from "../../Library/event-header/EventHeader";
+import EventDescription, {
+  EventDescriptionProps,
+} from "../../Library/event-description/EventDescription";
 
 type EventPageProps = {
   title: string;
   date: string;
   image: string;
-};
+} & EventDescriptionProps;
 
-const EventPage: FC<EventPageProps> = ({ title, date, image }) => {
+const EventPage: FC<EventPageProps> = ({
+  title,
+  date,
+  image,
+  descriptionTitle,
+  descriptionDescription,
+  horizontalTermLineData,
+  listDescriptionData,
+}) => {
   return (
     <article className="event-page">
       <EventHeader title={title} date={date} image={image} />
```

Comment:
Det her er ikke relateret til det her PR, men jeg så lige at, at der på EventHeader ikke bliver sendt en alt tekst med til billedet. 

Det tænker jeg at der burde, siden det ikke bare er et decorativt billede. 

Det kunne måske lige smides ind her, med et tag til den ticket hvor headeren blev lavet.

Reply:
Jeg tænker ikke, det er så relevant at kunne ændre al teksten fra Eventpage. Der er lavet en separat story for ImageCredited, hvor den kan ændres, hvis man har brug for at se det i design-systemet.

Men hvis I synes, det er vigtigt, kan jeg godt tilføje det?